torch/distributed/_shard/checkpoint/resharding.py

```diff
@@ -257,41 +262,57 @@ def _prepare_generic_tensor_read(
             )
     return read_reqs
 
-def _compute_tensor_md(storage_key: str, tensor: Tensor) -> TensorStorageMetadata:
+
+def _tensor_props_for(tensor: torch.Tensor) -> TensorProperties:
```

Comment:
I recalled we have `TensorProperties.create_from_tensor` added in previous PR? maybe we don't need this function.

Reply:
You're right, this is an oversight.